### PR TITLE
Added support for PowerStore v2.9.0

### PR DESCRIPTION
**Pull request by suryagupta4**
# Description
Added PowerStore v2.9.0 support and removed v2.6.0 config.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1012 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Ran make unit-test
- [x] Installation using the new sample tested and cert-csi executed.
- [x] Ran e2e for Powerstore test cases
image

### CODE REVIEW

**Comment by jooseppi-luna** [comment]
Can you confirm that you've tested installing with the sample file and run cert-csi against it? You can pull the nightly image for now since v2.9.0 has not been released yet.

**Comment by suryagupta4** [comment]
> Can you confirm that you've tested installing with the sample file and run cert-csi against it? You can pull the nightly image for now since v2.9.0 has not been released yet.

Yes. Installation with the new sample is done and executed cert-csi as well.

File: operatorconfig/moduleconfig/common/version-values.yaml [hunk 1]
```diff
@@ -37,6 +37,8 @@ powerstore:
       resiliency: "v1.6.0"
   v2.8.0:
       resiliency: "v1.7.0"
+  v2.9.0:
```
[the remark this answers is not in the document]
Comment (suryagupta4) [reply operatorconfig/moduleconfig/common/version-values.yaml:40]: @VamsiSiddu-7 Here we added Powerstore v2.9.0 entry only, module team (here Resiliency) will bump up the version in this file and add the necessary config files in moduleconfig directory.

[the remark this answers is not in the document]
Comment (VamsiSiddu-7) [reply operatorconfig/moduleconfig/common/version-values.yaml:40]: okay thanks surya

**Review by VamsiSiddu-7 — DISMISSED** [review]

File: operatorconfig/moduleconfig/common/version-values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (adarsh-dell) [reply operatorconfig/moduleconfig/common/version-values.yaml:40]: Certainly, the teams responsible for each module will increment the version numbers of their respective modules following the completion of validation processes.

**Review by adarsh-dell — DISMISSED** [review]

**Review by jooseppi-luna — COMMENTED** [review]
Thanks! Also realized, can we update the e2e test files as well, and execute the powerstore e2e tests to make sure they are good

**Comment by suryagupta4** [comment]
> Thanks! Also realized, can we update the e2e test files as well, and execute the powerstore e2e tests to make sure they are good

@jooseppi-luna Ran e2e with PowerStore test cases. Attached the result in the PR description.

**Review by jooseppi-luna — APPROVED** [review]
LGTM, thanks!

**Review by rishabhatdell — APPROVED** [review]

**Pull request merged**